docs/src/faqs.md

```diff
@@ -59,7 +59,7 @@ plot(sol; idxs = [A, B])
 ```
 
 ## How to disable rescaling of reaction rates in rate laws?
-As explained in the [Reaction rate laws used in simulations](@ref) section, for
```

Comment:
At least in the old docs this yields working links, however, Documenter produces a warning/error from it. writing out the reference, while annoying, removes this while preserving behaviours.